### PR TITLE
Route header and footer

### PR DESCRIPTION
**Pull request by iamsaurav28**

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+/***********/
+.header{
+  position: sticky;
+  top:0;
+  width:100%;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```
Comment (krishanu2911) [inline src/styles.css:92]: All these properties could be moved to utils

**Review by krishanu2911 — APPROVED** [review]
Everything is good to go no changes are required 
Just a suggestion to use utils CSS whenever possible other than that Good to go👍🏻

File: src/Components/Home.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+function Home() {
+  return (
+    <div>
+             <div class="row">
+               <div class="section">
+                    <h1>Get a Great Products<br/>In Best Budget</h1>
+                   
+                    <a href="#" class="btn">GET STARTED &#8594;</a>
```
Comment (Sameony) [inline src/Components/Home.js:10]: do replace the a tags with relative Link tags or div tags if it wasn;t meant to be

**Review by Sameony — APPROVED** [review]
the website is very smooth, good work on breaking down the components

File: src/styles.css (hunk 1 above)
Comment (iamsaurav28) [reply src/styles.css:92]: okay

File: src/Components/Home.js (hunk 2 above)
Comment (iamsaurav28) [reply src/Components/Home.js:10]: okk

Comment (iamsaurav28) [reply src/Components/Home.js:10]: no i m not gonna use that link  anywhere.

**Pull request merged**